server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/service/ExportServiceTest.java

```diff
@@ -0,0 +1,208 @@
+package org.apache.james.webadmin.service;
```

Comment:
We always prefix our files with the Apache license, that's a rule within the Apache foundation.